tests/integration/framework/process/daprd/daprd.go

```diff
@@ -327,17 +328,41 @@ func (d *Daprd) ProfilePort() int {
 	return d.profilePort
 }
 
+type MetricBucket struct {
+	UpperBound      float64
+	CumulativeCount uint64
+}
+
+type metricsOptions struct {
+	includeBuckets bool
+}
+
+func WithIncludeBuckets() func(metricsOptions *metricsOptions) {
+	return func(m *metricsOptions) {
+		m.includeBuckets = true
+	}
+}
+
 // Returns a subset of metrics scraped from the metrics endpoint
-func (d *Daprd) Metrics(t *testing.T, ctx context.Context) map[string]float64 {
+func (d *Daprd) Metrics(t *testing.T, ctx context.Context, opts ...func(*metricsOptions)) map[string]float64 {
 	t.Helper()
 
+	curOpts := metricsOptions{}
+	for _, o := range opts {
+		o(&curOpts)
+	}
```

Comment:
Is there a reason we should ever _not_ include the bucket metrics to the returned test func? Is this option really needed?

Reply:
not a big reason. the function description said a subset and why I added this as an optional one. 

I guess not including will save some memory and a bit of speed, even though for tests that might not be the most important thing.

I can have it always return it.

Reply:
@JoshVanL let me know your recommendation to change one way or another

Reply:
Let's always include all metrics in the output. Int tests are bound by `time.Sleep`s in Dapr runtime and test `eventually`s so we're not saving anything excluding them.

Reply:
done